x-pack/plugins/kubernetes_security/public/components/alerts_list/index.tsx

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { ReactNode } from 'react';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { EuiButtonIcon } from '@elastic/eui';
+import { CSSObject } from '@emotion/react';
+
+const alertsListContainer: CSSObject = {
```

Comment:
You might need to add `eui-yScroll` class to avoid the black scroll bar.
https://elastic.github.io/eui/#/utilities/scroll#vertical-scroll-y
![Screenshot 2022-05-25 at 17 39 38](https://user-images.githubusercontent.com/1490444/170303046-e219da42-49a5-401d-8202-d6b0ef7455bf.png)

Reply:
Hey @machadoum , the kubernetes page is behind a feature flag in `x-pack/plugins/security_solution/common/experimental_features.ts` with the `kubernetesEnabled` flag. When we set `kubernetesEnabled: true` it should be able to appear in the global search 
<img width="631" alt="image" src="https://user-images.githubusercontent.com/16872649/170311779-653f46c2-3090-4e9c-8765-97bbbc1ac1d3.png">

Thanks for pointing out adding the `eui-yScroll` class, I have updated it in my latest commit. Just for information - the `alerts_list` component along with other components in `x-pack/plugins/kubernetes_security/public/components` will need more changes in future works in the `kubernetes_security` plugin (since the UI/UX for the page isn't finalized yet). The current PR is mainly meant for the groundwork of the kubernetes page 😃